.github/workflows/ci-pipeline.yml

```
@@ -1,4 +1,4 @@
-name: xUnit Ext. CI/CD Pipeline
+name: xUnit Ext. CI Pipeline
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)